pandas/tools/merge.py

```diff
@@ -1436,13 +1438,90 @@ def concat(objs, axis=0, join='outer', join_axes=None, ignore_index=False,
     copy : boolean, default True
         If False, do not copy data unnecessarily
 
-    Notes
-    -----
-    The keys, levels, and names arguments are all optional
-
     Returns
     -------
     concatenated : type of objects
+
+    Notes
+    -----
+    The keys, levels, and names arguments are all optional.
+
+    Examples
+    --------
+    Combine two ``Series``.
+
+    >>> s1 = pd.Series(['a', 'b'])
+    >>> s2 = pd.Series(['c', 'd'])
+    >>> pd.concat([s1, s2])
+    0    a
+    1    b
+    0    c
+    1    d
+    dtype: object
+
+    Clear the existing index and reset it in the result
+    by setting the ``ignore_index`` option to ``True``.
+
+    >>> s1 = pd.Series(['a', 'b'])
+    >>> s2 = pd.Series(['c', 'd'])
+    >>> pd.concat([s1, s2], ignore_index=True)
+    0    a
+    1    b
+    2    c
+    3    d
+    dtype: object
+
+    Add a ``hierarchical index`` at the outermost level of
+    the data.
+
+    >>> s1 = pd.Series(['a', 'b', 'c'])
+    >>> s2 = pd.Series(['c', 'd', 'e'])
+    >>> c = pd.concat([s1, s2], keys=["s1", 's2',])
+    >>> c
+    s1  0    a
+        1    b
+        2    c
+    s2  0    c
+        1    d
+        2    e
+    dtype: object
+    >>> c.ix['s1']
```

Comment:
The above example is good. But I would leave out the examples on indexing the result to keep the examples focused on `concat` (such examples would certainly be nice for the tutorial docs)

Maybe add an example with `axis=1`